haddock/modules/scoring/default.py

```diff
@@ -51,11 +56,21 @@ def run(self, module_information):
         jobs = []
 
         # Get the models generated in previous step
-        models_to_score = [p for p in self.previous_io.output if p.file_type == Format.PDB]
+        models_to_score = []
```

Comment:
same as before.
if you want to be purist, these logic should be some kind of method of function. but for now just keep the list comprehension